testutil/network/sessionnet/genesis.go

```diff
@@ -0,0 +1,109 @@
+package sessionnet
+
+import (
+	"fmt"
+	"testing"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/stretchr/testify/require"
+
+	apptypes "github.com/pokt-network/poktroll/x/application/types"
+	sharedtypes "github.com/pokt-network/poktroll/x/shared/types"
+	suppliertypes "github.com/pokt-network/poktroll/x/supplier/types"
+)
+
+// configureDefaultSupplierModuleGenesisState generates and populates the in-memory
+// network's application module's GenesisState object with a given number of suppliers,
+// each of which is staked for a unique service. It returns the genesis state object.
+func (memnet *inMemoryNetworkWithSessions) configureSupplierModuleGenesisState(t *testing.T) *suppliertypes.GenesisState {
+	t.Helper()
+
+	require.NotEmptyf(t, memnet.GetNetworkConfig(t), "cosmos config not initialized, call #Start() first")
+	require.NotEmptyf(t, memnet.PreGeneratedAccountIterator, "pre-generated accounts not initialized, call #Start() first")
+
+	// Create NumSuppliers number of suppliers with a service "svcX", where "X" is the supplier's index.
+	var supplierGenesisState = suppliertypes.DefaultGenesis()
+	for i := 0; i < memnet.Config.NumSuppliers; i++ {
+		preGenerateAcct, ok := memnet.PreGeneratedAccountIterator.Next()
+		require.True(t, ok)
+
+		supplier := sharedtypes.Supplier{
+			Address: preGenerateAcct.Address.String(),
+			Stake:   &sdk.Coin{Denom: "upokt", Amount: sdk.NewInt(10000)},
+			Services: []*sharedtypes.SupplierServiceConfig{
+				{
+					Service: &sharedtypes.Service{Id: fmt.Sprintf("svc%d", i)},
+					Endpoints: []*sharedtypes.SupplierEndpoint{
+						{
+							Url:     "http://localhost:1",
+							RpcType: sharedtypes.RPCType_JSON_RPC,
+						},
+					},
+				},
+			},
+		}
+		supplierGenesisState.SupplierList = append(supplierGenesisState.SupplierList, supplier)
+	}
+
+	supplierGenesisBuffer, err := memnet.GetNetworkConfig(t).Codec.MarshalJSON(supplierGenesisState)
+	require.NoError(t, err)
+
+	// Add supplier module genesis supplierGenesisState to the network config.
+	memnet.GetNetworkConfig(t).GenesisState[suppliertypes.ModuleName] = supplierGenesisBuffer
+
+	return supplierGenesisState
+}
+
+// TODO_IMPROVE: draw a mermaid diagram of the app/supplier/service session network.
+
+// configureDefaultSupplierModuleGenesisState generates and populates the in-memory
```

Comment:
Does this comment belong to this method?

Reply:
Is this from dead code 👍🏼 to what @red-0ne said

Reply:
:pray: